userspace/libsinsp/sinsp.cpp

```diff
@@ -504,6 +515,18 @@ void sinsp::fill_tp_of_interest(scap_open_args *oargs, const std::unordered_set<
 	}
 }
 
+std::unordered_set<uint32_t> sinsp::get_all_tp()
```

Comment:
`get_tracepoint_names` vs `get_all_tp` vs `enforce_sinsp_state_tracepoints` vs (private)`fill_tp_of_interest` :D

Reply:
:thinking: 
 It's really hard to follow that. :exploding_head:

Reply:
yeah I will rename all of them with the `tp` suffix